lib/standards/index.js

```diff
@@ -0,0 +1,31 @@
+import ariaAttrs from './aria-attrs';
+import { clone } from '../core/utils';
+
+const origAriaAttrs = clone(ariaAttrs);
+const standards = {
+	ariaAttrs
```

Comment:
For safety, I think we should freeze the contents of these things:

```suggestion
	ariaAttrs: deepFreeze(ariaAttrs)
```

Reply:
If we deep freeze the object then we can't add in new properties. I cloned the original so we could override `ariaAttrs` without worries.